test/launchesReporting.test.js

```diff
@@ -23,6 +23,7 @@ jest.mock('./../lib/utils', () => ({
     name: 'agentName',
     version: 'agentVersion',
   }),
+  parseAttributes: () => [],
```

Comment:
Why are you mocking this value? Can it be tested too with launch start?

Reply:
As I investigated, we don't test specific attributes values

Reply:
But it would be good to test it here (not specific values, but different formats).
So when the launch will be started in the project configured via `mocharc.json`, we expect that the attributes will be reported in the right way.
It would be great to add one more test for `launch start` for this case.